src/runtime_src/core/tools/xbmgmt2/flash/xspi.cpp

```diff
@@ -32,6 +32,7 @@
 
```

Comment:
Copyright date (line 2) needs to be updated.

Reply:
comment addressed.